src/NClient/NClient.Standalone/ClientProxy/Generation/MethodBuilders/Providers/TimeoutAttributeProvider.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using NClient.Annotations;
+using NClient.Core.Mappers;
+using NClient.Standalone.Exceptions.Factories;
+
+namespace NClient.Standalone.ClientProxy.Generation.MethodBuilders.Providers
+{
+    internal interface ITimeoutAttributeProvider
+    {
+        ITimeoutAttribute? Get(MethodInfo method, IEnumerable<MethodInfo> overridingMethods);
```

Comment:
There is an implicit convention - if a method has a possibility of not finding an attribute and returning null, then it should have the `Find` name.

Reply:
Ok.